### ISSUE TITLE
Cant Edit Existing Alert

### ISSUE THREAD

**Issue by benny-tam** [metabase/metabase#10999]
**Describe the bug**
After I created an alert for certain questions, 
it seems that it is impossible to alter the alert anymore. 

Every time when I go to that alert and edit it, 
the page will show a new popup for it. 
However this popup will close as soon as I click on any place of the page.

**Comment by flamber** [comment]
Hi @benny-tam 
Please provide the information requested, when creating an issue, like version, error logs, steps-to-reproduce.

**Comment by benny-tam** [comment]
Logs: 
```
[6181cd9b-eef9-4fd3-9682-1f6cb699526a] 2019-09-27T14:01:02+08:00 DEBUG metabase.middleware.log GET /api/database 200 5.9 ms (3 DB calls) Jetty threads: 3/50 (4 idle, 0 queued) (63 total active threads) Queries in flight: 0
[6181cd9b-eef9-4fd3-9682-1f6cb699526a] 2019-09-27T14:01:02+08:00 DEBUG metabase.middleware.log GET /api/setup/admin_checklist 200 21.2 ms (11 DB calls) Jetty threads: 3/50 (4 idle, 0 queued) (63 total active threads) Queries in flight: 0
[6181cd9b-eef9-4fd3-9682-1f6cb699526a] 2019-09-27T14:01:02+08:00 DEBUG metabase.middleware.log GET /api/setting 200 442.7 µs (0 DB calls) Jetty threads: 5/50 (2 idle, 0 queued) (63 total active threads) Queries in flight: 0
[6181cd9b-eef9-4fd3-9682-1f6cb699526a] 2019-09-27T14:01:02+08:00 DEBUG metabase.middleware.log GET /api/activity/recent_views 200 6.5 ms (1 DB calls) Jetty threads: 5/50 (2 idle, 0 queued) (63 total active threads) Queries in flight: 0
[6181cd9b-eef9-4fd3-9682-1f6cb699526a] 2019-09-27T14:01:02+08:00 DEBUG metabase.middleware.log GET /api/activity 200 35.8 ms (6 DB calls) Jetty threads: 5/50 (2 idle, 0 queued) (65 total active threads) Queries in flight: 0
[6181cd9b-eef9-4fd3-9682-1f6cb699526a] 2019-09-27T14:01:08+08:00 DEBUG metabase.middleware.log GET /api/session/properties 200 3.2 ms (1 DB calls) Jetty threads: 3/50 (4 idle, 0 queued) (67 total active threads) Queries in flight: 0
[6181cd9b-eef9-4fd3-9682-1f6cb699526a] 2019-09-27T14:01:08+08:00 DEBUG metabase.middleware.log GET /api/setup/admin_checklist 200 20.6 ms (11 DB calls) Jetty threads: 3/50 (4 idle, 0 queued) (67 total active threads) Queries in flight: 0
[6181cd9b-eef9-4fd3-9682-1f6cb699526a] 2019-09-27T14:01:09+08:00 DEBUG metabase.middleware.log GET /api/setting 200 507.0 µs (0 DB calls) Jetty threads: 3/50 (4 idle, 0 queued) (67 total active threads) Queries in flight: 0
[6181cd9b-eef9-4fd3-9682-1f6cb699526a] 2019-09-27T14:01:09+08:00 DEBUG metabase.middleware.log GET /api/task 200 5.8 ms (2 DB calls) Jetty threads: 3/50 (4 idle, 0 queued) (67 total active threads) Queries in flight: 0
[6181cd9b-eef9-4fd3-9682-1f6cb699526a] 2019-09-27T14:03:06+08:00 DEBUG metabase.middleware.log GET /api/user/current 200 10.8 ms (3 DB calls) Jetty threads: 3/50 (4 idle, 0 queued) (63 total active threads) Queries in flight: 0
[6181cd9b-eef9-4fd3-9682-1f6cb699526a] 2019-09-27T14:03:06+08:00 DEBUG metabase.middleware.log GET /api/session/properties 200 3.1 ms (1 DB calls) Jetty threads: 3/50 (4 idle, 0 queued) (63 total active threads) Queries in flight: 0
[6181cd9b-eef9-4fd3-9682-1f6cb699526a] 2019-09-27T14:03:06+08:00 DEBUG metabase.middleware.log GET /api/database 200 6.6 ms (3 DB calls) Jetty threads: 3/50 (4 idle, 0 queued) (63 total active threads) Queries in flight: 0
[6181cd9b-eef9-4fd3-9682-1f6cb699526a] 2019-09-27T14:03:07+08:00 DEBUG metabase.middleware.log GET /api/setting 200 524.3 µs (0 DB calls) Jetty threads: 3/50 (4 idle, 0 queued) (63 total active threads) Queries in flight: 0
[6181cd9b-eef9-4fd3-9682-1f6cb699526a] 2019-09-27T14:03:11+08:00 DEBUG metabase.middleware.log GET /api/pulse/form_input 200 113.8 ms (0 DB calls) Jetty threads: 3/50 (4 idle, 0 queued) (63 total active threads) Queries in flight: 0
[6181cd9b-eef9-4fd3-9682-1f6cb699526a] 2019-09-27T14:03:15+08:00 DEBUG metabase.middleware.log GET /api/user/current 200 6.2 ms (3 DB calls) Jetty threads: 4/50 (3 idle, 0 queued) (63 total active threads) Queries in flight: 0
[6181cd9b-eef9-4fd3-9682-1f6cb699526a] 2019-09-27T14:03:15+08:00 DEBUG metabase.middleware.log GET /api/session/properties 200 4.2 ms (1 DB calls) Jetty threads: 4/50 (3 idle, 0 queued) (63 total active threads) Queries in flight: 0
[6181cd9b-eef9-4fd3-9682-1f6cb699526a] 2019-09-27T14:03:16+08:00 DEBUG metabase.middleware.log GET /api/user/current 200 5.9 ms (3 DB calls) Jetty threads: 3/50 (4 idle, 0 queued) (63 total active threads) Queries in flight: 0
[6181cd9b-eef9-4fd3-9682-1f6cb699526a] 2019-09-27T14:03:16+08:00 DEBUG metabase.middleware.log GET /api/database 200 6.0 ms (3 DB calls) Jetty threads: 3/50 (4 idle, 0 queued) (63 total active threads) Queries in flight: 0
[6181cd9b-eef9-4fd3-9682-1f6cb699526a] 2019-09-27T14:03:16+08:00 DEBUG metabase.middleware.log GET /api/setting 200 528.1 µs (0 DB calls) Jetty threads: 3/50 (4 idle, 0 queued) (63 total active threads) Queries in flight: 0
```

```
dataset::fieldRefForColumn mismatch
app-main.bundle.js?a0cb8a0cb1424da110c7:5 dataset::fieldRefForColumn mismatch
app-main.bundle.js?a0cb8a0cb1424da110c7:5 dataset::fieldRefForColumn mismatch
app-main.bundle.js?a0cb8a0cb1424da110c7:5 dataset::fieldRefForColumn mismatch
app-main.bundle.js?a0cb8a0cb1424da110c7:5 dataset::fieldRefForColumn mismatch
app-main.bundle.js?a0cb8a0cb1424da110c7:5 dataset::fieldRefForColumn mismatch
app-main.bundle.js?a0cb8a0cb1424da110c7:5 dataset::fieldRefForColumn mismatch
app-main.bundle.js?a0cb8a0cb1424da110c7:5 dataset::fieldRefForColumn mismatch
app-main.bundle.js?a0cb8a0cb1424da110c7:5 dataset::fieldRefForColumn mismatch
app-main.bundle.js?a0cb8a0cb1424da110c7:5 dataset::fieldRefForColumn mismatch
11:1 Refused to execute script from 'http://libblerdatateam.us-east-1.elasticbeanstalk.com/snippets/text.js' because its MIME type ('text/html') is not executable, and strict MIME type checking is enabled.
```

**To Reproduce**
1. go to an existing question (with existing alert)
2. click on alert, and click on edit
3. click on anywhere of the page
4. the popup is closed immediately

**Expected behavior**
- The popup should not be closed till user save / cancel the editing OR close the popup

**Information about your Metabase Installation:**

- chrome: Version 77.0.3865.90 (Official Build) (64-bit)
- OS: Windows 10
- Databases: Postgres
- Metabase: version v0.33.2
- hosting environment: Elastic Beanstalk
- internal database: H2 
-

**Comment by flamber** [comment]
@benny-tam 
I cannot reproduce "click anywhere of the page" and the popup will close on 0.33.2, but there was a couple of fixes to the popup in 0.33.3, so it now fills the entire page and doesn't let you click top bar.

**Comment by benny-tam** [comment]
did not notice there is a new version.. 
let me try and update it first.
will revisit the ticket if this issue still occurred
